packages/gestalt/src/Tabs.js

```diff
@@ -11,10 +11,24 @@ type OnChangeHandler = ({|
   +activeTabIndex: number,
 |}) => void;
 
+function Circle() {
+  return (
+    <Box
+      color="red"
+      dangerouslySetInlineStyle={{ __style: { marginTop: '1px' } }}
```

Comment:
@danignibus could we avoid the 1px margin on the top - since it's a magic number which we want to avoid. Please check with design, if they require the `1px`, let's leave it in.

![image](https://user-images.githubusercontent.com/127199/86626679-e0b40c00-bf7b-11ea-8b32-41bd318b7cff.png)
![image](https://user-images.githubusercontent.com/127199/86626779-07724280-bf7c-11ea-9c73-7d36b9c9e7d9.png)